contracts/SignatureMerkleDrop128.sol

```diff
@@ -42,6 +42,7 @@ contract SignatureMerkleDrop128 is ISignatureMerkleDrop128, Ownable {
         if (!valid) revert InvalidProof();
         _invalidate(index);
         IERC20(token).safeTransfer(receiver, amount);
+        payable(receiver).sendValue(msg.value);
```

Comment:
Let’s make it conditional? If msg.value > 0

Reply:
f6bf753